src/helpers.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+declare -r BASHUNIT_GIT_REPO="https://github.com/TypedDevs/bashunit"
```

Comment:
```suggestion
declare -r BASHUNIT_GIT_REPO="git@github.com:TypedDevs/bashunit.git"
```

Reply:
This will improve the speed. Good catch!